test/bin/build_images.sh

```diff
@@ -534,15 +555,26 @@ mkdir -p "${IMAGEDIR}/blueprints"
 mkdir -p "${IMAGEDIR}/builds"
 mkdir -p "${VM_DISK_BASEDIR}"
 
-configure_package_sources
-
 # Prepare container lists for mirroring registries.
 rm -f "${CONTAINER_LIST}"
 extract_container_images "${SOURCE_VERSION}" "${LOCAL_REPO}" "${CONTAINER_LIST}"
 extract_container_images "4.${FAKE_NEXT_MINOR_VERSION}.*" "${NEXT_REPO}" "${CONTAINER_LIST}"
 extract_container_images "4.${FAKE_YPLUS2_MINOR_VERSION}.*" "${YPLUS2_REPO}" "${CONTAINER_LIST}"
 extract_container_images "${PREVIOUS_RELEASE_VERSION}" "${PREVIOUS_RELEASE_REPO}" "${CONTAINER_LIST}"
 
+# Original values of CURRENT_RELEASE_REPO and PREVIOUS_RELEASE_REPO are needed for proper
```

Comment:
This is OK for now. When we have the current issue resolved, we may want to consider using separate variables for the "real" repos vs. the pre-release content from the mirror. It means more variables, but the logic using those variables can be more explicit.

Reply:
PTAL at most recent force push and see if it's better